apps/core/scenes/search/PlacePicker/PlaceItem.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  Touchable,
+} from '@kiwicom/universal-components';
+
+import { type Location } from '../SearchContext';
+
+type Props = {|
+  +onPress: Location => void,
+  +location: ?Location,
+|};
+
+class PlaceItem extends React.Component<Props> {
+  handlePress = () => {
+    const { location, onPress } = this.props;
+    location && onPress(location); // eslint-disable-line babel/no-unused-expressions
+  };
+
+  render() {
+    const { location } = this.props;
+    if (!location || !location.locationId || !location.name) {
+      // eslint-disable-next-line no-console
+      console.error('Incomplete data for the location!');
```

Comment:
This will cause red-screen and probably crash in production build. `console.warn` is probably better, though I am not sure we need any console.log for this 🤔 

also the if-condition can be improved by `!location?.locationId || !location?.name)`

Reply:
Mine idea behind this was about made a log somehow if this happens.


> also the if-condition can be improved by `!location?.locationId || !location?.name)` 
	
Thanks, I always forget about this new feature.

Reply:
done